stable/kubecost/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
-appVersion: 1.84.0
+appVersion: 1.84.1
```

Comment:
@faiq (lmk if I should stop pinging you about these things 😆 ) can we get the script to update `appVersion` to the chart version? Or where does this script live again, maybe I can take a look?